src/subcommand/import.rs

```diff
@@ -70,7 +71,11 @@ fn import_line(db: &mut Db, line: &str) -> Result<()> {
         .parse::<f64>()
         .with_context(|| format!("invalid rank: {}", rank_str))?;
 
-    let path = util::canonicalize(&path)?;
+    let path = if config::zo_resolve_symlinks() {
```

Comment:
`import_line()` is run once for each line in the input. To avoid parsing `_ZO_RESOLVE_SYMLINKS` every time, I suggest you pass a bool `resolve_symlinks` from the `import()` function.

Reply:
Done, thanks, I didn't look how this function was used.